api/pom.xml

```diff
@@ -376,35 +368,18 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>3.1.0</version>
                     <configuration>
-                        <failOnError>true</failOnError>
+                        <source>8</source>
                         <doctitle>Jakarta JSON Binding ${project.version} API Specification</doctitle>
-                        <javaApiLinks>
-                            <property>
-                                <name>api_1.3</name>
-                                <value>http://download.oracle.com/javase/1.3/docs/api/</value>
-                            </property>
-                            <property>
-                                <name>api_1.4</name>
-                                <value>http://download.oracle.com/javase/1.4.2/docs/api/</value>
-                            </property>
-                            <property>
-                                <name>api_1.5</name>
-                                <value>http://download.oracle.com/javase/1.5.0/docs/api/</value>
-                            </property>
-                            <property>
-                                <name>api_1.6</name>
-                                <value>http://download.oracle.com/javase/6/docs/api/</value>
-                            </property>
-                            <property>
-                                <name>api_1.7</name>
-                                <value>http://download.oracle.com/javase/7/docs/api/</value>
-                            </property>
-                            <property>
-                                <name>api_1.8</name>
-                                <value>http://download.oracle.com/javase/8/docs/api/</value>
-                            </property>
-                        </javaApiLinks>
+                        <sourceFileExcludes>
+                            <sourceFileExclude>**/module-info.java</sourceFileExclude>
+                            <sourceFileExclude>target/**/*.java</sourceFileExclude>
+                        </sourceFileExcludes>
```

Comment:
why is this?

Reply:
Javadoc version 8 is used ('<source>8</source>'). It doesn't understand module-info.java

Reply:
so the same sourcefile exclusion defined at https://github.com/eclipse-ee4j/jsonb-api/pull/161/files#diff-8658bb96fff6ac848cd28d3242fce067R177 is not relevant?